data/json/furniture_and_terrain/furniture-fungal.json

```diff
@@ -61,5 +61,33 @@
     "required_str": -1,
     "flags": [ "FLAMMABLE_ASH", "FUNGUS" ],
     "bash": { "str_min": 26, "str_max": 50, "sound": "poof.", "sound_fail": "squelch." }
+  },
+  {
+    "type": "terrain",
+    "id": "t_fungus_colony",
+    "name": "mushroom colony",
+    "description": "A large pile of organic matter which has molded over with fungus.  The caps of many mushrooms spring from the top, all of which are fortunately terrestrial. You could snip some of the mushrooms from it.",
+    "symbol": "#",
+    "color": "dark_gray",
+    "move_cost": 30,
```

Comment:
This is most likely an error. AFAIK no terrain in the game has this much `move_cost`. If you want to make it very hard to traverse, make it around 8, which is A LOT.

Reply:
I changed it from 30 to 2 on both furniture tiles.